modules/exploits/windows/http/lg_simple_editor_rce.rb

```diff
@@ -0,0 +1,188 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::EXE
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper # includes register_files_for_cleanup
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'LG Simple Editor Remote Code Execution',
+        'Description' => %q{
+          This Metasploit module exploits broken access control and directory traversal
+          vulnerabilities in LG Simple Editor software for gaining code execution.
+          The vulnerabilities exists in versions of LG Simple Editor prior to v3.21.
```

Comment:
I saw the ZDI advisory/ the references listed below only mention one CVE but here you're mentioning the module exploits multiple vulnerabilities. Should there be another CVE listed in the references? Just curious. 
```suggestion
          The vulnerabilities exist in versions of LG Simple Editor prior to v3.21.
```

Reply:
Yes, module exploit two vulns, but directory traversal is the only one reported to ZDI. The second vuln is broken access control because the `uploadImage.do` endpoint does not require authentication. I discovered the second vulnerability while trying to exploit the first one. I couldn't find any CVE or related publication; It might be a 0day.

Reply:
Cool, after a quick search I couldn't find anything related to the second vuln either. There were 25 CVEs disclosed in the security bulletin and ZDI specified that the path traversal is exploitable without authentication. I say we land this as is for now, however when more info comes out if this does turn out to be an 0day we can help get you a CVE assigned.